### PR TITLE
WordPress.com logotype and the WP .org logo in login screen

### LINKED ISSUE #4742: Change logo to WordPress.com when logging in to WordPress.com
To match https://github.com/wordpress-mobile/WordPress-iOS/pull/6142

We'll switch the logo based on what log in screen the user is currently in

### PR DESCRIPTION
**Pull request by hypest**
Fixes #4742 

To test:
* Clear the app data
* Open the app. The WordPress.com logo and logotype will be displayed at the top
* Tap on the "Add self-hosted site" text-button
* The screen changes into one close to that on the https://login.wordpress.org/ site
* Tapping the "Log in to WordPress.com" text-button goes back to the blue WordPress.com mode

Here's a video of the transition: https://cloudup.com/cpdZrbd7dRd

### CODE REVIEW

File: WordPress/src/main/java/org/wordpress/android/ui/accounts/SignInFragment.java [hunk 1]
```diff
@@ -1,16 +1,21 @@
 package org.wordpress.android.ui.accounts;
 
+import android.animation.Animator;
```
Comment (nbradbury) [inline WordPress/src/main/java/org/wordpress/android/ui/accounts/SignInFragment.java]: These three imports are unused:
* android.animation.Animator
* android.animation.AnimatorInflater
* android.support.transition.Transition

File: WordPress/src/main/res/values/styles.xml [hunk 2]
```diff
@@ -190,6 +190,13 @@
         <item name="android:clickable">true</item>
     </style>
 
+    <style name="WordPress.NUXFlatButtonNoBg">
+        <item name="android:textColor">@color/grey_lighten_10</item>
+        <item name="android:background">?android:attr/selectableItemBackground</item>
+        <item name="android:padding">12dp</item>
```
Comment (nbradbury) [inline WordPress/src/main/res/values/styles.xml]: Minor nit: this can be `@dimen/margin_large` rather than `12dp`.

**Comment by nbradbury** [comment]
@hypest Apart from two very minor things, I think this looks good. I'm not convinced it will help with .com vs .org confusion, but regardless it's a nice effect and I think it's still an improvement :)

**Comment by hypest** [comment]
Thanks for review Nick, ready for another pass!

> I'm not convinced it will help with .com vs .org confusion, but regardless it's a nice effect and I think it's still an improvement :)

No problem! I too think this won't be a big help in general, but the extra clues can help _some_ users and hopefully will do :)

For the record, let me also share some info about the history of this PR/approach. So, I set out to fix the original issue https://github.com/wordpress-mobile/WordPress-Android/issues/4742 while it was still planned for the 6.2 milestone. It seemed simple to add the WordPress.com logotype anyway. While doing it though, it occurred to me that by adding the logotype, it made the .org case look a bit wrong on the branding side since the logo (simplified WordPress logo) was not really the one .org uses. You can probably see the rabbit hole opening right there ;)

This PR is the result of making the two modes more correctly branded and give some extra clues to the user to identify the different login inputs.

**Comment by nbradbury** [comment]
:shipit:

**Pull request merged**